docs/application/native/guides/notification/attach-panel.md

```diff
@@ -123,7 +134,7 @@ To create an attach panel:
     s_info.attach_panel = attach_panel;
     ```
 
-2. Based on the type of content you want the user to be able to select for the attach panel, add content categories using the `attach_panel_add_content_category()` function. The available content categories are defined in the [attach_panel_content_category](../../api/mobile/latest/group__CAPI__PANEL__ATTACH__MODULE.html#gada3a2db6ac8e7d42b7dff7c3cc48720b)enumerator.The content categories in the **More** tab are shown in the frequency, recently used, and alphabetical sequence.To deliver more information to the UI gadget or called application, add the data with a bundle.
+2. Select the type of content for the attach panel. Add content categories using the `attach_panel_add_content_category()` function. The available content categories are defined in the [attach_panel_content_category](../../api/mobile/latest/group__CAPI__PANEL__ATTACH__MODULE.html#gada3a2db6ac8e7d42b7dff7c3cc48720b) enumerator. The content categories in the **More** tab are shown in the frequency of recently used and alphabetical sequence. To deliver more information to the UI gadget or called application, add the data within a bundle.
```

Comment:
@hallower 
Hello,
 According to me,  [attach_panel_content_category] enumeration is a collection of different enumerators like ATTACH_PANEL_CONTENT_CATEGORY_IMAGE, ATTACH_PANEL_CONTENT_CATEGORY_CAMERA, ATTACH_PANEL_CONTENT_CATEGORY_VOICE, and so on. In that case, Please change the word enumerator to enumeration.

Reply:
Thank you for comment.
Would you let me know what this mean "Please change the word enumerator to enumeration."?

Reply:
The available content categories are defined in the [attach_panel_content_category](../../api/mobile/latest/group__CAPI__PANEL__ATTACH__MODULE.html#gada3a2db6ac8e7d42b7dff7c3cc48720b) enumerator. 
In the sentence, attach_panel_content_category is an enumeration, but in the sentence it is written as enumerator.

Reply:
Dear @Reni-90 
I will raise new pull request for that.